app/views/newContributionsTest.scala.html

```diff
@@ -0,0 +1,71 @@
+@import assets.AssetsResolver
```

Comment:
this is just a copy of the non "Test" one with an extra parameter and calling test.scala.html instead of main.scala.html.